plugins/flashrom/fu-flashrom-device.c

```diff
@@ -30,6 +42,16 @@ fu_flashrom_device_init (FuFlashromDevice *self)
 	fu_device_add_icon (FU_DEVICE (self), "computer");
 }
 
+static void
+fu_flashrom_device_finalize (GObject *object)
+{
+	FuFlashromDevicePrivate *priv = GET_PRIVATE (FU_FLASHROM_DEVICE (object));
+	G_OBJECT_CLASS (fu_flashrom_device_parent_class)->finalize (object);
```

Comment:
I think the G_OBJECT_CLASS() is normally last, no? I don't know if it matters, but I think it's destroying the parent object and so the FuFlashromDevicePrivate would be invalid.

Reply:
Done, Thanks!